src/track/beatutils.cpp

```diff
@@ -5,10 +5,10 @@
  *      Author: vittorio
  */
 
-#include <QtDebug>
-#include <QString>
 #include <QList>
 #include <QMap>
+#include <QString>
+#include <QtDebug>
```

Comment:
Does clang actually sort includes or is this a leftover manual change?

Reply:
Includes are sorted, default option of the google style.